tech.json

```diff
@@ -1318,6 +1318,15 @@
             "Being able to 4 tap is expected as that brings out the most value from this tech."
           ]
         },
+        {
+          "name": "canSpikeSuit",
+          "requires": [{"gainFlashSuit": {}}],
+          "note": [
+            "The ability to gain a flash suit using spikes or similar damage source.",
+            "Gain a shinecharge, take a spike hit while morphed, unmorphing either 1 or 2 frames after taking damage.",
```

Comment:
spike, thorn, electricity etc

I was reading the wiki and was under the impression you unmorph on frame 0 or 1, and then the spark is activated on frame 10 after the spike, independent of when you unmorph (so 9-10 after the unmorph)

Reply:
The Practice Hack [docs](https://smpractice.speedga.me/infohudmode) give more detail about this so I based the description off of that:

> E1 indicates you unmorphed the same frame you took damage, which is one frame early.

I suppose this could still be consistent with the Wiki if damage happens one frame after contact? I'm not really sure.